spanner/spannertest/README.md

```diff
@@ -24,6 +24,7 @@ by ascending esotericism:
 - SELECT HAVING
 - more literal types
 - DEFAULT
+- THEN RETURN
```

Comment:
We are supporting it through this PR, right? I guess we want to say PG.Returning is not supported.

Reply:
No, I did not intend to add this support myself. We will be adding support to the standard Spanner Emulator however.